domains/dbauch.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Personal GitHub",
+  "repo": "https://github.com/bauchdj",
+  "owner": {
+    "username": "is-a-dev",
+    "email": "involved.worm.wqdi@mask.me"
+  },
+  "record": {
+    "CNAME": "https://github.com/bauchdj"
```

Comment:
Invalid CNAME. CNAME cannot contain protocols, file paths, URL parameters, or anything combination of those. See docs for more information: https://docs.is-a.dev/